src/main/java/seedu/address/ui/popupwindow/AddCommandPopupWindow.java

```diff
@@ -139,6 +152,15 @@ public void setCloseWindowKey(KeyCode keyCode) {
         });
     }
 
+    /**
+     * Refreshes the current list.
+     */
+    private void refresh() {
```

Comment:
I see how you do the refreshing of the display list.
An alternative is to keep some Boolean values in this class and switch accordingly, but his solution is better. Good job!